src/test/java/seedu/address/logic/commands/CancelClearCommandTest.java

```diff
@@ -0,0 +1,38 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.TypicalRecipes.getTypicalRecipeBook;
+
+import org.junit.jupiter.api.Test;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for.
+ * {@code ClearCommand}.
+ */
+public class CancelClearCommandTest {
+    private final Model model = new ModelManager(getTypicalRecipeBook(), new UserPrefs());
+    private final Model expectedModel = new ModelManager(getTypicalRecipeBook(), new UserPrefs());
+
+    @Test
+    public void execute_showsCancellationMessage() {
```

Comment:
Naming suggestion: can add the expected value for the test at the end of the test name

```suggestion
    public void execute_showsCancellationMessage_success() {
```